src/styles/_scripts.scss

```diff
@@ -0,0 +1,17 @@
+/* ================ 3rd Party Script styling overrides ================ */
+
+/**
+ * Needed to make sure cookie banner doesn't have white text on white background
+ * (Cards for the website are configured to have white background, but body text is also configured to be white).
+ * stylelint-disable-next-line selector-max-id
+ */
+#CybotCookiebotDialog {
+    color: $dark-9;
+    opacity: 0.95;
+    z-index: 30 !important;
+}
+
+/* Show one prompt at a time. Cookie Banner covers ChatBot until complete */
+#drift-frame-controller {
+    z-index: 5 !important;
+}
```

Comment:
In VSCode under settings, search for `Files: Insert Final Newline` so that on save it automatically adds a new line at the end of a file for you 🤓

Reply:
✨